### PR TITLE
Bump ansible to 2.15 & enable content-actions

### PR DESCRIPTION
**Pull request by Ruchip16**
Bumps ansible to 2.15 and enables content-actions

### CODE REVIEW

File: tox-ansible.ini [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[ansible]
+
+skip =
+    py3.7
+    py3.8
+    2.9
+    2.10
+    2.11
+    2.12
+    2.13
```
Comment (KB-perByte) [inline tox-ansible.ini]: This is not needed

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   release:
     types: [published]
 
 jobs:
   release_automation_hub:
-    uses: ansible/devtools/.github/workflows/release_ah.yml@main
+    uses: ansible/ansible-content-actions/.github/workflows/release.yaml@main
     with:
-      environment: ah
+      environment: release
```
Comment (KB-perByte) [inline .github/workflows/release.yml]: why did we add this change?

File: tox-ansible.ini (hunk 1 above)
Comment (Ruchip16) [reply tox-ansible.ini]: the entire file is not needed ?

Comment (KB-perByte) [reply tox-ansible.ini]: We still have this file in a lot of our collections we can remove it, anyways it is outdated

**Review by KB-perByte — CHANGES_REQUESTED** [review]

**Review by NilashishC — APPROVED** [review]

File: tox-ansible.ini (hunk 1 above)
Comment (NilashishC) [reply tox-ansible.ini]: doesn't tox-ansible complain if you don't have this file at all? maybe keep it as a placeholder with skip = "" for skips we may add in the future?

Comment (Ruchip16) [reply tox-ansible.ini]: that makes sense to me

**Review by roverflow — APPROVED** [review]

**Review by KB-perByte — APPROVED** [review]

**Pull request merged**